docker/Dockerfile.aarch64-unknown-linux-gnu

```diff
@@ -30,13 +30,21 @@ RUN /linux-image.sh aarch64
 
 COPY linux-runner base-runner.sh /
 
+COPY toolchain.cmake /opt/toolchain.cmake
+ENV CMAKE_TOOLCHAIN_FILE=/opt/toolchain.cmake
```

Comment:
We can use a single CMake toolchain file for every target besides Android. The `CROSS_CMAKE_OBJECT_FLAGS` are provided so the same flags are used by default when CMake is used outside of `cmake-rs`, for whatever reason. `cmake-rs` will override these, so it's not a major deal.